.env

```diff
@@ -1 +1,2 @@
-REACT_APP_API_URL=/
+REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
```

Comment:
This should be just `REACT_APP_API_URL=/` because we are now using the proxy configuration.